x-pack/plugins/observability/public/components/app/observability_status/observability_status_boxes.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { ObservabilityStatusBox } from './observability_status_box';
+
+export interface ObservabilityStatusProps {
+  boxes: Array<{
+    id: string;
+    dataSource: string;
+    hasData: boolean;
+    description: string;
+    modules: Array<{ name: string; hasData: boolean }>;
+    integrationLink: string;
+    learnMoreLink: string;
+  }>;
+}
+
+export function ObservabilityStatusBoxes({ boxes }: ObservabilityStatusProps) {
+  return (
+    <div>
+      {boxes.map((box) => (
+        <ObservabilityStatusBox key={box.id} {...box} />
```

Comment:
I noticed the `*Box` components have a `style={{ marginBotton: "20px" }}`. I think it's better to use a `EuiSpacer` here. That way we reuse the theme margin sizes.

```
{boxes.map((box) => (
  <>
    <ObservabilityStatusBox key={box.id} {...box} />
    <EuiSpacer />
  </>
))}
```